src/realm/cluster.cpp

```diff
@@ -972,8 +972,8 @@ void Cluster::nullify_incoming_links(ObjKey key, CascadeState& state)
 {
     size_t ndx = get_ndx(key, 0);
     if (ndx == realm::npos)
-        throw KeyNotFound(util::format("When nullify incoming links for key '%1' in '%2'", key.value,
-                                       get_owning_table()->get_name()));
+        throw KeyNotFound(util::format("Key '%1' not found in '%2' when nullifying incoming links", key.value,
```

Comment:
This exception message was written with the assumption that the exception name would be printed before it, and was very confusing without that.